src/layouts/Settings/Settings.tsx

```diff
@@ -34,11 +36,13 @@ import { SocialMediaSettings } from "./SocialMediaSettings"
 
 export const Settings = (): JSX.Element => {
   const { siteName } = useParams<{ siteName: string }>()
+  const { userId } = useLoginContext()
+  const isGithubUser = userId !== "Unknown user" && !!userId
```

Comment:
shall we add a todo here as well? Since this would prob have to be modified as this is not a real `isGithubUser`?

Reply:
What do you mean by this sorry? `userId` is a field that's only present on github login (or people not logged in completely)